driver/src/test/java/org/neo4j/driver/internal/value/NodeValueTest.java

```diff
@@ -70,12 +70,12 @@ public void shouldTypeAsNode()
         assertThat( value.typeConstructor(), equalTo( TypeConstructor.NODE ) );
     }
 
-    private NodeValue emptyNodeValue()
+    public static NodeValue emptyNodeValue()
```

Comment:
Please extract `NodeValueTest#emptyNodeValue()`, `PathValueTest#pathValue()` and `RelationshipValueTest#emptyRelationshipValue()` into a test utility class. Alternatively `ValuesTest` could just create it's own node, relationship and path values.